### PR TITLE
Upgrade to Angular 10

### PR DESCRIPTION
**Pull request by Aracturat**
Upgrade this library to support Angular 10. Please publish it after accept.

### CODE REVIEW

File: projects/angular-iban/package.json [hunk 1]
```diff
@@ -16,9 +16,11 @@
     "typescript"
   ],
   "peerDependencies": {
-    "@angular/common": "~9.1.11",
-    "@angular/core": "~9.1.11",
+    "@angular/common": "^10.x",
+    "@angular/core": "^10.x",
     "iban": "0.0.14"
   },
-  "license": "MIT"
```
Comment (WuglyakBolgoink) [inline projects/angular-iban/package.json:23]: @Aracturat Tnx, но "license" нужна для npm-registry.

**Review by WuglyakBolgoink — CHANGES_REQUESTED** [review]

File: projects/angular-iban/package.json (hunk 1 above)
Comment (Aracturat) [reply projects/angular-iban/package.json:23]: Уппс, случайно снес, сейчас верну

**Comment by Aracturat** [comment]
@WuglyakBolgoink Fixed license file

**Comment by Aracturat** [comment]
@fundsaccess can you look at this PR?

**Comment by fundsaccess** [comment]
Hello @Aracturat,

thanks for your PR.  We will review it and give you feedback.

Kind regards,
fundsaccess F/E team

**Review by fundsaccess — APPROVED** [review]
Hi @Aracturat ,

thank you so much for this PR. It's looking very good. Just a suggestion for the future, before you would contribute to this project, please create an issue and let's talk about it at first. Minor fixes or improvements that don't have with the upgrade to do should be handled in separate issues and PRs. 

We are looking forward to your next contribution! 

Kind regards,
fundsaccess F/E Team

**Pull request merged**